dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.ts

```diff
@@ -72,7 +72,7 @@ export const getAmendmentsByFacilityId = async (req: Request, res: Response) =>
       default:
         if (amendmentIdOrStatus) {
```

Comment:
not your problem, but we could guard clause here and remove a layer of indentation

Reply:
This is a case where I'd end up refactoring the whole controller which I want to avoid 😅